src/link/rgblink.1

```diff
@@ -113,6 +122,45 @@ When making a ROM, be careful that not using this is not a replacement for
 .Xr rgbfix 1 Ap s Fl p
 option!
 .El
+.Ss Scrambling algorithm
+The default section placement algorithm tries to minimize the amount of banks used;
```

Comment:
```suggestion
The default section placement algorithm tries to minimize the number of banks used.
```

Reply:
The two propositions together compare the two algorithms, so I don't think a period is good style.

Reply:
Semicolons are fine then.